python/setup.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/bash
+
+# Updates the OS
+sudo apt update && sudo apt -y upgrade
+
+# Creates the directory where the repo will get cloned
+sudo mkdir /opt/starflit
+
+# Changes ownership of the starflit directory
+sudo chown ${USER} /opt/starflit 
+
+# Clones the starflit repository
+git clone https://github.com/lbo462/starflit.git /opt/starflit
+
+# Creates a virtual env
+virtualenv --system-site-packages /opt/starflit/starflit/python/venv
+
+# Activates the virtual env
+source /opt/starflit/starflit/python/venv/bin/activate
```

Comment:
Instead of using source, maybe one could directly use the location of pip in the venv, it would be safer